components/Dialogs/src/ColorThemer/MDCAlertColorThemer.m

```diff
@@ -21,6 +21,13 @@
 
 @implementation MDCAlertColorThemer
 
++ (void)applySemanticColorScheme:(nonnull id<MDCColorScheming>)colorScheme
+               toAlertController:(nonnull MDCAlertController *)alertController {
+  alertController.titleColor = [colorScheme.onSurfaceColor colorWithAlphaComponent:(CGFloat)0.87];
```

Comment:
I would expect the alpha to already be part of onSurfaceColor.
If this is not the case, we should consider documenting that in another PR.

Reply:
Unfortunately this isn't the case - opacity is often used to augment state independent from the color system.